core/domain/prod_validators_test.py

```diff
@@ -381,6 +381,26 @@ def test_standard_operation(self):
         self.run_job_and_check_output(
             expected_output, sort=False, literal_eval=False)
 
+    def test_model_with_commiter_id_migration_bot(self):
```

Comment:
Add tests for all the models that now allow the new IDs.

Reply:
Done.